.travis.yml

```diff
@@ -15,9 +15,6 @@ php:
   - hhvm
 
 matrix:
-  include:
-    - php: 5.3
-      env: deps=low
```

Comment:
The code must work with lowest deps on PHP 5.3. Just omitting some tests just to make the build pass is not the right solution.